src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -499,6 +505,9 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
         Kind.EXPECT_CLASS -> require(funSpec.body.isEmpty()) {
           "functions in expect classes can't have bodies"
         }
+        Kind.EXTERNAL_CLASS -> require(funSpec.body.isEmpty()) {
```

Comment:
Same here, you can merge this case with `Kind.EXPECT_CLASS` and print `$kind` instead of `expect` and `external`

Reply:
done